.github/workflows/trigger_docs_deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Triggers docs deployment workflow
+
+on:
+  workflow_run:
+    workflows:
+      - compress
+    types:
+      - completed
+    branches:
+      - main
+
+jobs:
+  trigger_docs_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Triggers docs deployment workflow
+        env:
+          PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
+        run: |
+          curl -X POST https://api.github.com/repos/GetStream/stream-chat-docusaurus/actions/workflows/docusaurus-build.yml/dispatches \
+          -H 'Accept: application/vnd.github.v3+json' \
+          -u ":$PAT_TOKEN" \
```

Comment:
@vishalnarkhede you probably want do replace the current secret with your PAT_TOKEN instead of mine.

This is the only permission you need:
![image](https://user-images.githubusercontent.com/18744505/119733531-10982680-be50-11eb-8c84-d8259a6f3b31.png)